packages/js/product-editor/src/components/checkbox-control/checkbox-control.tsx

```diff
@@ -60,9 +60,6 @@ export const Checkbox: React.FC< CheckboxProps > = ( {
 						position="top center"
 						// eslint-disable-next-line @typescript-eslint/ban-ts-comment
 						// @ts-ignore Incorrect types.
-						className={
-							'woocommerce-product-form__checkbox-tooltip'
-						}
```

Comment:
I removed this prop because it seems like [it's not being used](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/tooltip/index.tsx#L28-L34) in the Tooltip component.